app/views/posts/index.html.erb

```diff
@@ -1,6 +1,33 @@
-<h2>List of posts for User <%= @user.name %></h2>
-<ul>
+<div class="posts-container">
+  <div class="user-header">
+    <div class="user-image">
+      <%= image_tag @user.photo, alt: @user.name, class: "profile-photo" %>
+    </div>
+    <div class="user-info">
+      <h2><%= @user.name %></h2>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+
   <% @posts.each do |post| %>
-    <li><%= link_to post.title, user_post_path(@user, post) %></li>
+    <div class="post-card">
+      <h3><%= post.title %></h3>
+      <p><%= post.body.truncate(100) %></p>
+      <div class="post-stats">
+        <span>Comments: <%= post.comments_counter %></span>
+        <span>Likes: <%= post.likes_counter %></span>
+      </div>
+      <div class="post-comments">
+        <% post.comments.each do |comment| %>
```

Comment:
* Also here According to the wireframe you are to have a method that returns 5 most recent posts of the user so in your post index view file you will render the comments like this ``<% post.recent_comments.each do |comment| %>`` 
![image](https://github.com/cancelei/blog-in-rails/assets/52497246/d635a73d-5d23-4a5a-a7d8-32a3ca66dc6b)